tests/unit/writers/qbwriter/test_qbwriter.py

```diff
@@ -199,7 +200,10 @@ def test_csv_col_required():
         csv_col = empty_qbwriter._generate_csvw_column_definition(col)
         required = csv_col["required"]
         assert isinstance(required, bool)
-        assert required
+        if isinstance(col.structural_definition, QbAttribute):
+            assert required == False
```

Comment:
This should only happen in a cube with multiple obs val columns. This cube has only one obs-val column. And perhaps more concerningly, this isn't even a pivoted shape cube.

Reply:
Robsays hi